tests/nfa/nfa.cc

```diff
@@ -733,25 +734,25 @@ TEST_CASE("Mata::Nfa::construct() invalid calls")
 	{
 		parsec.type = "FA";
 
-		CHECK_THROWS_WITH(construct(parsec),
-			Catch::Contains("expecting type"));
+		CHECK_THROWS_WITH(Builder::construct(parsec),
+                          Catch::Contains("expecting type"));
 	}
 
 	SECTION("construct() call with an epsilon transition")
 	{
 		parsec.type = Mata::Nfa::TYPE_NFA;
 		parsec.body = { {"q1", "q2"} };
 
-		CHECK_THROWS_WITH(construct(parsec),
-			Catch::Contains("Epsilon transition"));
+		CHECK_THROWS_WITH(Builder::construct(parsec),
+                          Catch::Contains("Epsilon transition"));
 	}
 
 	SECTION("construct() call with a nonsense transition")
 	{
 		parsec.type = Mata::Nfa::TYPE_NFA;
 		parsec.body = { {"q1", "a", "q2", "q3"} };
 
-		CHECK_THROWS_WITH(construct(&aut, parsec),
+		CHECK_THROWS_WITH(Plumbing::construct(&aut, parsec),
```

Comment:
It is little weird that `construct` is in two namespaces. Is this intended or an error?

Reply:
It is intended. The one in `Mata::Nfa::Plumbing` namespace is the hidden pointer-to-nfa kind of function, which all reside in plumbing namespace. This was here before as well. Only now instead of `Mata::Nfa::construct()`, the normal functions are in `Mata::Nfa::Builder` namespace.